src/LogMonitor.js

```diff
@@ -256,13 +258,13 @@ export default class LogMonitor extends Component {
           <LogMonitorButton
```

Comment:
I'd suggest to hide this button when `importStorage` prop is not specified (since it is not required). Probably:
```js
{importStorage && <LogMonitorButton ... />}
```

And the same for `exportStorage` bellow.